### PR TITLE
Refactor a vector test and make sanitizer tests more accessible

### PR DESCRIPTION
**Pull request by AdityaAtulTewari**

### CODE REVIEW

File: .github/workflows/drivex.yml [hunk 1]
```diff
@@ -56,6 +55,13 @@ jobs:
         echo "PANDO_TEST_DISCOVERY_TIMEOUT=600" >> $GITHUB_ENV
         echo "IMAGE_VERSION=$(git log --pretty="%h" -1 Dockerfile.dev)" >> $GITHUB_ENV
         echo "PANDO_CONTAINER_ENV=-e=PANDORT_TESTS=ON" >> $GITHUB_ENV
+        echo ${{ runner.name }}
+        if [ ${{ runner.name }} == 'zerberus-0' ]; then
```
Comment (patrickkenney9801) [inline .github/workflows/drivex.yml:59]: I thought there were 4 runners?

Comment (patrickkenney9801) [reply .github/workflows/drivex.yml:59]: Also does zerberus have hyperthreads enabled?

**Review by patrickkenney9801 — APPROVED** [review]

File: .github/workflows/drivex.yml (hunk 1 above)
Comment (AdityaAtulTewari) [reply .github/workflows/drivex.yml:59]: no right now we are doing 2.

Comment (AdityaAtulTewari) [reply .github/workflows/drivex.yml:59]: yes hyperthreads are enabled, but these splice the sockets

**Pull request merged**